samples/monster_generated.h

```diff
@@ -25,6 +25,14 @@ enum Color {
   Color_MAX = Color_Blue
 };
 
+inline Color (&EnumValuesColor())[3] {
```

Comment:
Both usages are applied in cases where an enumeration is Sparse, as you mentioned at the start.

There is some assurance provided to a developer that the above for each syntax will continue to operate, even if another developer adds an enumeration value, regardless of that value. Our game uses this to output the valid inputs in an console client.